patches/openwrt/0007-xrx200-ensure-the-correct-version-for-fritz7360.patch

```diff
@@ -0,0 +1,94 @@
+From fa3af8c6583b5338499f10aec926f49d399e84ee Mon Sep 17 00:00:00 2001
+From: Grische <github@grische.xyz>
+Date: Sun, 18 Sep 2022 14:03:16 +0200
+Subject: [PATCH] xrx200: ensure the correct version for fritz7360
+
```

Comment:
Please add a Link to this PR in the patch description, so it is possible to correlate it later down the road.

Reply:
Added